V1Michigan/v1
# UMich OAuth login

### Status
**READY**

![image](https://user-images.githubusercontent.com/7119822/158045620-f0921296-ccc8-4179-8fed-474c0049cbc0.png)

### Description
* Remove email/password for authentication, just use OAuth
* Combine `Login` and `SignUp`, since the code to sign up or log in with OAuth is the same
* Remove `/join` + `SignUp`
* Add the UMich logo to `/login` to make it clear users should use their `@umich.edu` account

Feel free to comment on design + copy, but we'll probably make some changes to it tomorrow either way

### Migrations
NO

### Steps to Test or Reproduce

```sh
git pull --prune
git checkout umich-oauth-login
npm run dev
```

1. Navigate to `/login`. Make sure you can still sign up (as a new user) and log in (with an existing user) correctly
2. Feel free to comment on design + copy, but we'll probably make some changes to it tomorrow either way


Thanks! :smiley:
